x/gov/common_test.go

```diff
@@ -16,7 +16,7 @@ import (
 )
 
 var (
-	valTokens           = sdk.TokensFromConsensusPower(42)
+	valTokens           = sdk.TokensFromConsensusPower(42, sdk.DefaultPowerReduction)
```

Comment:
Recommend an update to `defaultValTokens`

Reply:
I know its not new to this PR, but could there be a comment explaining what valTokens is as well? My understanding is that its `// The default power validators are initialized to have within tests`